{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/base.html

```diff
@@ -5,8 +5,8 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>{% block title %}{% endraw %}{{ cookiecutter.project_name }}{% raw %}{% endblock title %}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="">
+    <meta name="description" content="{{ cookiecutter.description }}">
+    <meta name="author" content="{{ cookiecutter.author_name }}">
```

Comment:
The build failed because it's a Django template, but as cookiecutter uses Jinja under the hood, the project is itself a Jinja template. This file is therefore mixing Jinja + Django and Jinja processing is generally disabled with `raw` tag.

Here we want to inject data coming from cookiecutter, in Jinja processing should be enabled temporarily like in line 6:

```suggestion
    <meta name="description" content="{% endraw %}{{ cookiecutter.description }}{% raw %}">
    <meta name="author" content="{% endraw %}{{ cookiecutter.author_name }}{% raw %}">
```